invenio_app_rdm/records_ui/views/records.py

```diff
@@ -195,9 +198,12 @@ def record_detail(
     # NOTE: this should maybe be an expandable field instead
     record_owner = record._record.parent.access.owner.resolve()
     resolved_community, _ = get_record_community(record_ui)
-    theme = (
-        resolved_community.to_dict().get("theme", {}) if resolved_community else None
+    resolved_community = (
+        UICommunityJSONSerializer().dump_obj(resolved_community.to_dict())
```

Comment:
This is similar to what happens in the deposit page: https://github.com/inveniosoftware/invenio-app-rdm/blob/4bfc51dc6d1fa3e5e1de20a2805e1ee287432e60/invenio_app_rdm/records_ui/views/decorators.py#L361